tools/src/lib.rs

```diff
@@ -43,3 +43,8 @@ const ENV: &str = r#"
 NEXUS_VM_K=16
 NEXUS_VM_PROVER=nova-seq
 "#;
+
+#[cfg(feature = "verbose")]
+pub(crate) const TERMINAL_MODE: nexus_tui::Mode = nexus_tui::Mode::Enabled;
+#[cfg(not(feature = "verbose"))]
+pub(crate) const TERMINAL_MODE: nexus_tui::Mode = nexus_tui::Mode::Disabled;
```

Comment:
"verbose" feature is not defined as the cli is always verbose. `tools` use `TerminalHandle::new_enabled()`

Reply:
Fixed.